goji/commands.py

```diff
@@ -308,13 +308,25 @@ def search(client, format, query):
     issues = client.search(query)
 
     for issue in issues:
-        click.echo(format.replace('\\n', '\n').format(
+        # LKBM
```

Comment:
What does LKBM stand for?

Reply:
Luca Kaceem Butler Masters. Was just a "hey, this is where I made changes" comment for myself. :-)